### ISSUE TITLE
Add all Amp Models

### ISSUE THREAD

**Issue by dbagchee** [dbagchee/helix-preset-viewer#1]
Add full amp model list to hxModels.

**Comment by maikito26** [comment]
This one doesn't seem fun.  I assume we'll need several patches to extract this info due to DSP limitations.  That said, I can't extract these myself until my HX Stomp arrives.  Same goes for Amps.

**Comment by dbagchee** [comment]
Yep - it will be painful - I'll have to see if there is any other way to extract the model names en masse.

Same for #16 - there are tons of unique parameters across all the models so it will be time consuming to harvest them all.

**Comment by splawrence** [comment]
Can you just use the model names from the patch notes form Line 6? I would love to create a PR to update this but I also don't have a Helix atm.
